examples/events_custom_state_event/system.rs

```diff
@@ -0,0 +1,18 @@
+use crate::event::MyCustomGameEventType;
+use amethyst::{
+    core::shrev::EventChannel,
+    derive::SystemDesc,
+    ecs::{System, SystemData, Write},
+};
+
+#[derive(SystemDesc)]
+pub(crate) struct SpammingSystem;
+
+impl<'a> System<'a> for SpammingSystem {
+    type SystemData = Write<'a, EventChannel<MyCustomGameEventType>>;
+
+    fn run(&mut self, mut my_event_channel: Self::SystemData) {
+        my_event_channel.single_write(MyCustomGameEventType::MyEnumVariant);
+        println!("Sending event");
```

Comment:
I suggest we illustrate this a different way, without spamming the tty

Reply:
Ok, I just copied what was being done in the other event-example, do you have an idea of something better? :)

Reply:
I think we could do it with Ui interactions for triggering events, and signalling their receipt. But this is just my preference, im sure it’s fine to land without that.

Reply:
I made some changes, let me know what you think :)

Reply:
It’s ok to me, I think it could be better if the event was triggered by some interactive action, and signaled by something in the window. But I can respect keeping it simple, too. LGTM